openhtf/__init__.py

```diff
@@ -525,10 +525,15 @@ def __call__(self, test_state):
     kwargs = dict(self.extra_kwargs)
     kwargs.update(test_state.plug_manager.provide_plugs(
         (plug.name, plug.cls) for plug in self.plugs if plug.update_kwargs))
-    arg_info = inspect.getargspec(self.func)
+    try:
+      arg_info = inspect.getfullargspec(self.func)
```

Comment:
Maybe put this behind a hasattr check instead of catching the AttributeError.

Reply:
Or maybe a version check?
`if sys.version_info[0] < 3:`
That make it more clear why this is needed.